warehouse/packaging/tasks.py

```diff
@@ -95,7 +96,7 @@ def metadata_backfill_individual(request, file_id):
         return
 
     base_url = request.registry.settings.get("files.url")
-    file_url = base_url.format(path=file_.path)
+    file_url = urllib.parse.quote(base_url.format(path=file_.path), safe=":/")
```

Comment:
Probably fine for the backfill, but we've been trying to only use one url parser, and settled on urllib3 - see https://github.com/pypi/warehouse/pull/14497

Reply:
AFAICT, I'm not actually seeing a way to do this with `urllib3`. Given that we're going to be deleting this task in ~1 week, I'm just going to merge this as is!